src/scenarios/personalization/ProgressSet/ProgressSet.tsx

```diff
@@ -0,0 +1,181 @@
+              Authorization: `Bearer ${userAccessToken}`,
+            },
+          },
+          fetchPolicy: 'no-cache',
+        });
+
+        logger.log(`method [${setProgress.name}]`, 'output:', result.data);
+
+        if (result.errors) {
+          logger.error(result.errors);
+        }
+      } catch (error) {
+        if (error instanceof Error) {
+          logger.error(
+            `method [${setProgress.name}]`,
+            'output:',
+            error.message,
+          );
+        }
+      }
+    }, FrequencyOfProgressSaving);
+
+    setIntervalId(handler);
+  }
+
+  return (
+    <>
+      <Segment basic>
+        <Header size="huge">Progress</Header>
+        <Header size="small">
+          Required Services:
+          <Label>user-service</Label>
+          <Label>personalization-service</Label>
+        </Header>
+
+        <Divider />
+
+        <Container fluid>
+          <p>
+            This scenario demonstrates the main use case for saving the
+            user&apos;s progress of the video playback.
+          </p>
+        </Container>
+
+        <Divider />
+
+        <Grid divided>
+          <Grid.Column width={5}>
+            <Form>
+              <Form.Input
+                control={VariableSearch}
+                label="Frequency of progress saving (ms)"
+                value={FrequencyOfProgressSaving}
+                setStateValue={setFrequencyOfProgressSaving}
+              />
+
+              <Form.Input
+                control={VariableSearch}
+                icon="key"
+                label="User Access Token"
+                value={userAccessToken}
+                setStateValue={setUserAccessToken}
+              />
+
+              <Form.Input
+                control={VariableSearch}
+                label="Key"
+                value={key}
+                setStateValue={setKey}
+              />
+
+              <Button toggle active={buttonState} onClick={setProgress}>
+                {buttonState === true ? 'Run' : 'Pause'}
+              </Button>
+            </Form>
+          </Grid.Column>
+          <Grid.Column width={5}>
+            <div
+              style={{
+                fontSize: '16px',
+                lineHeight: 2,
+                paddingLeft: 100,
+                paddingTop: 10,
+                textAlign: 'center',
+                fontWeight: 'bold',
+              }}
+            >
+              <div className="stopwatch">
+                <div className="numbers">
+                  <span>
+                    {('0' + Math.floor((time / 3600000) % 60)).slice(-2)}:
+                  </span>
+                  <span>
+                    {('0' + Math.floor((time / 60000) % 60)).slice(-2)}:
+                  </span>
+                  <span>
+                    {('0' + Math.floor((time / 1000) % 60)).slice(-2)}:
+                  </span>
+                  <span>{('0' + ((time / 10) % 100)).slice(-2)}</span>
```

Comment:
Do we need to show milliseconds (10ms blocks in this case) in the timer? Last time we discussed to only show down to seconds. Did anything change in this decision? If not, let's get rid of this part and only have 1 second resolution in the timer